templates/http/base/rhoai/notebook.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: kubeflow.org/v1
+kind: Notebook
+metadata:
+  annotations:
+    notebooks.opendatahub.io/inject-oauth: 'true'
+    opendatahub.io/image-display-name: Minimal Python
+    openshift.io/display-name: {{values.name}}
+  name: {{values.name}}
+  namespace: {{values.dsp}}
+  labels:
+    app: {{values.name}}
```

Comment:
do we need this?  or the parent app is set already by `base/kustomization.commonLabels`

Reply:
Huh, TIL, if it's set by the common labels, then no, it's not needed.

Reply:
Updated to remove the label